hs_core/hydroshare/resource.py

```diff
@@ -301,11 +305,35 @@ def check_resource_type(resource_type):
         raise NotImplementedError("Type {resource_type} does not exist".format(resource_type=resource_type))
     return res_cls
 
+def add_zip_file_contents_to_resource_async(resource, f):
+    """
+    Launch asynchronous celery task to add zip file contents to a resource.
+    Note: will copy the zip file into a temporary space accessible to both
+    the Django server and the Celery worker.
+    :param resource: Resource to which file should be added
+    :param f: TemporaryUploadedFile object (or object that implements temporary_file_path())
+     representing a zip file whose contents are to be added to a resource.
+    """
+    # Add contents of zipfile asynchronously; wait 30 seconds to be "sure" that resource creation
+    # has finished.
+    logger = logging.getLogger('django')
```

Comment:
Move creation of the logger to the module level and use `getLogger(__name__)` so that messages are identified by the module from which they originate.